### PR TITLE
third-parties-cat

### PR DESCRIPTION
**Pull request by erwinpalma**
third-parties-cat

See the issue  #26123 

# CLOSE|Close #[*issue_number Short description*]
[*Long description*]

# NEW|New [*Short description*]
[*Long description*]

### CODE REVIEW

**Comment by erwinpalma** [comment]
Thas is my fisrt contribution, I hope that I have done ti well

**Comment by eldy** [comment]
You should implement a pre commit git hook so your code will be formated correctly.
See https://wiki.dolibarr.org/index.php?title=Environment_and_development_tools#Install_Pre-commit_git_hook
Currently your code submission generates a lot of noise diff that make not possible to compare your change...

**Comment by erwinpalma** [comment]
> You should implement a pre commit git hook so your code will be formated correctly. See https://wiki.dolibarr.org/index.php?title=Environment_and_development_tools#Install_Pre-commit_git_hook Currently your code submission generates a lot of noise diff that make not possible to compare your change...

I have followed the steps described in the document, thanks for the info @eldy

File: htdocs/societe/card.php [hunk 1]
```diff
@@ -1842,25 +1842,26 @@ function formatCustomerSelection (selection) {
...
-			print '<tr class="visibleifcustomer"><td class="toptd">'.$form->editfieldkey('CustomersProspectsCategoriesShort', 'custcats', '', $object, 0).'</td><td colspan="3">';
-			$cate_arbo = $form->select_all_categories(Categorie::TYPE_CUSTOMER, null, 'parent', null, null, 1);
-			print img_picto('', 'category', 'class="pictofixedwidth"').$form->multiselectarray('custcats', $cate_arbo, GETPOST('custcats', 'array'), null, null, 'quatrevingtpercent widthcentpercentminusx', 0, 0);
-			print "</td></tr>";
-
-			if (!empty($conf->global->THIRDPARTY_SUGGEST_ALSO_ADDRESS_CREATION)) {
-				print '<tr class="individualline"><td class="toptd">'.$form->editfieldkey('ContactCategoriesShort', 'contcats', '', $object, 0).'</td><td colspan="3">';
-				$cate_arbo = $form->select_all_categories(Categorie::TYPE_CONTACT, null, 'parent', null, null, 1);
-				print img_picto('', 'category', 'class="pictofixedwidth"').$form->multiselectarray('contcats', $cate_arbo, GETPOST('contcats', 'array'), null, null, 'quatrevingtpercent widthcentpercentminusx', 0, 0);
+			if ($object->client != 0) {
```
Comment (eldy) [inline htdocs/societe/card.php:1845]: The view of the category should already be managed by the class="visibleifcustomer"
So when we modify the status of thirdparty from non customer to customer, the javascript is able to automatically make the line with the selection of the category visible with no need to save and go back on form.

Comment (erwinpalma) [reply htdocs/societe/card.php:1845]: Hi @eldy,

Yes, that is the error I want to fix.

**Pull request closed without merging**